metron-platform/metron-parsers/src/test/resources/org/apache/metron/parsers/cef/cyberark.json

```diff
@@ -0,0 +1,21 @@
+{
+    "\"Other info\"": "101.198.70.93",
+    "\"Safe Name\"": "Security Vulnerability Mgmt",
+    "\"Ticket Id\"": "Needed to verify config files being pulled ",
+    "deviceAction": "Retrieve password",
+    "deviceAddress": "120.99.70.3",
```

Comment:
There is no IP parsing. IPs are purely represented as strings in Metron, so this shouldn't be an issue. This would only be an issue at the validation stage, which is out of scope here.